### DISCUSSION TITLE
Problem with assign a role in KeyCloak unable to give user readwrite access

### DISCUSSION THREAD

**Discussion by tmckiknn3** [keycloak/keycloak#31348] [Q&A]
I am looking to give a user read-write access, but when I do, the user does not have the OTP option on the sign-access. I have done this for myself and other users, but this user ID is the only one 
it will not work with. I have also tried deleting and recreating the ID. I have attached a screenshot for review.

**Comment by danielFesenmeyer** [comment]
Are you sure you have attached the screenshot?
And what do you mean with "read-write" access, and which role is causing the problem?

Furthermore it would be helpful to have a description of the expected outcome and the actual outcome.
